src/components/pages/dashboard/dashboard.js

```diff
@@ -66,38 +72,41 @@ export class Dashboard extends Component {
     // Telemetry stream - START
     const onPendingStart = () => this.setState({ telemetryIsPending: true });
 
-    const telemetry$ = TelemetryService.getTelemetryByDeviceIdP15M()
+    const getTelemetryStream = ({ deviceIds = [] }) => TelemetryService.getTelemetryByDeviceIdP15M(deviceIds)
       .merge(
         this.telemetryRefresh$ // Previous request complete
-          .delay(Config.telemetryRefreshInterval) // Wait to refresh
+          .delay(200000 || Config.telemetryRefreshInterval) // Wait to refresh
```

Comment:
Shouldn't the config value come first.  Then fallback to a hardcoded default if not configured?

Reply:
Woops, left a temp value in there